startreedata/thirdeye
# [notification] Make timers dynamic per namespace

#### Issue(s)

[Metrics - per namespace](https://startree.atlassian.net/browse/TE-2454)

#### Description

This PR adds add a namespace tag to all metrics where relevant

#### Testing

- [X] Existing UTs
- [X] Existing Integration tests

1. Run local thirdeye server and UI
2. Put some load on metrics by creating few alerts and notification groups
3. check different metrics by using following command
`curl localhost:8090/admin/prometheus | grep your_metric`

[local-server-prom-logs.txt](https://github.com/user-attachments/files/17929888/local-server-prom-logs.txt)
